packages/stream-metadata/src/routes/userTips.ts

```diff
@@ -0,0 +1,95 @@
+import { FastifyReply, FastifyRequest } from 'fastify'
+import { StreamPrefix, StreamStateView, makeStreamId } from '@towns-protocol/sdk'
+import { z } from 'zod'
+
+import { getStream } from '../riverStreamRpcClient'
+import { isValidEthereumAddress } from '../validators'
+
+const paramsSchema = z.object({
+	userId: z.string().min(1, 'userId parameter is required'),
+})
+
+export interface UserTips {
+	sent: Record<string, string>
+	received: Record<string, string>
+	sentCount: Record<string, string>
+	receivedCount: Record<string, string>
```

Comment:
what exactly is the sent / received field?
a map that stores what? userId? eth value? 

how about tip currency?

Reply:
got it! thought it was userId 😅